SaaS REST store code clarification (#412)
AdobeDocs/commerce-extensibility: src/pages/app-development/extension-compatibility.md

    + commerceGot(`V1/oope_payment_method/`, {
    ```

> - Ensure that store view codes are no longer included URLs. Instead, you need to specify the store views in the `Store` header of the request.

Reviewer: ```suggestion
- Ensure that store view codes are no longer included in URLs. Instead, you must specify store views in the `Store` header of the request.
```
